.github/pull_request_template.md

```diff
@@ -1,26 +1,12 @@
-# Objective
-<!-- Describe what this change achieves, and the details of how it works. -->
-
-# Motivation and Context
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to the issue here. -->
-
-# Screenshots (if appropriate)
-
-# Checklist
-<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-- [ ] If this contains visual changes, has it been reviewed by a designer?
-- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
-- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
-- [ ] I have or will communicate these changes to the front end practice
-
-# Additional Considerations
-- Does there need to be right-to-left (RTL) options for localization and internationalization?
-- Has the test suite been updated?
-- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
-- Have you updated any relevant documentation or left useful comments in the code?
-- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
-- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
-- Have Storybook stories been updated?
-
-**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.
+<!-- Hi there! 👋 -->
+<!-- If you are unsure about contributing to Kaizen, -->
+<!-- please reach out to us in #prod_design_systems on Slack. -->
+<!-- We are always happy to help you with any questions.  -->
```

Comment:
IMO this should be the repo index README rather than in a PR template.

Along the same lines of why the current one doesn't work well—by this time, the person has already done work and about to create a PR, so it's not like they're just landing here for the first time and trying to work out how to contribute. Whereas the repo README is a likely first place for someone to land for the first time and want some beginner guidance on how to contribute.

Reply:
Yes, you're right - I'll remove it and create a card to review the README as a next step.